libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -292,6 +292,7 @@ dictionary LnPaymentDetails {
     string payment_preimage;
     boolean keysend;
     string bolt11;
+    string? payer_bolt11;
```

Comment:
NIT: Since we have decided to expose the implementation details here perhaps the name "open_channel_bolt11" better fits here? I am thinking also of this field as the one that distinguish jit payments from the other.